src/test/kotlin/org/rust/ide/template/postfix/IntoIterPostfixTemplateTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.template.postfix
+
+import com.intellij.testFramework.LightProjectDescriptor
+import org.rust.WithStdlibRustProjectDescriptor
+
+
+class IntoIterPostFixTemplateTest : PostfixTemplateTest(IntoIterPostfixTemplate(RsPostfixTemplateProvider())) {
+    override fun getProjectDescriptor(): LightProjectDescriptor = WithStdlibRustProjectDescriptor
+
+    fun testNonIterable() = doTestNotApplicable(
```

Comment:
We usually use the following code stuly for tests
```kotlin
fun `test name of test case`() = someIdentifier("""
    code before
""", """
    code after (if applicable)
""")
```

Yeah, there are test cases with old naming but we will convert them eventually